@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -1184,7 +1213,7 @@ export class MapControls extends THREE.EventDispatcher {
                 const pinchDistance = this.calculatePinchDistanceInWorldSpace();
                 this.moveAlongTheViewDirection(pinchDistance);
             } else if (this.mapView.projection.type === geoUtils.ProjectionType.Spherical) {
-                // TODO: HARP-6597: Implement rotation for globe.
+                // TODO: HARP-6597: Implement yaw rotation for globe.
                 const pinchDistance = this.calculatePinchDistanceInWorldSpace();
                 this.setZoomLevel(this.currentZoom + pinchDistance * 0.01);
```

Comment:
Would love to have this as some static const definition, maybe PINCH_ZOOM_LEVEL_RATIO ?

Reply:
Right, however this is only for the in progress work on the controls. Later the use of `moveAlongTheViewDirection` will be implemented in lieu of `setZoomLevel` when zooming with touch, as in flat version above.